spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-polling-sample/src/main/java/com/example/SinkExample.java

```diff
@@ -33,7 +32,6 @@
  *
  * @since 1.2
  */
-@EnableBinding(PollableSink.class)
```

Comment:
Is this no longer needed? The documentation should be updated as well.

Reply:
The legacy samples are deleted, with docs updated.